ui-tests/tests/search.spec.ts

```diff
@@ -71,3 +71,28 @@ test('should get no matches', async ({ page }) => {
     await page.waitForSelector('#jp-search-replace >> text="No Matches Found"')
   ).toBeTruthy();
 });
+
+test('should test for case sensitive option', async ({ page }) => {
```

Comment:
Could you add test to the tow other buttons please?

Reply:
done